lib/elephant_in_the_room/sites/post.ex

```diff
@@ -156,21 +163,40 @@ defmodule ElephantInTheRoom.Sites.Post do
   end
 
   def get_or_insert_tag(name, site_id) do
-    inserted_tag = Repo.insert(%Tag{name: name, site_id: site_id, color: "686868"},
-      [on_conflict: :nothing])
+    inserted_tag =
+      Repo.insert(
+        %Tag{name: name, site_id: site_id, color: "686868"},
+        on_conflict: :nothing
+      )
```

Comment:
disable autoformarter for code unrelated to a commit

Reply:
Ok!